services/notify/src/irc.js

```diff
@@ -1,12 +1,26 @@
 const debug = require('debug')('notify');
-const Promise = require('bluebird');
-const irc = Promise.promisifyAll(require('irc-upd'));
+const util = require('util');
+const irc = require('irc-upd');
 const assert = require('assert');
 const aws = require('aws-sdk');
 const _ = require('lodash');
 
 const MAX_RETRIES = 5;
 
+Object.keys(irc).forEach( key => {
```

Comment:
I think I'd rather see us directly wrap things in promises below or avoid using promises when they are unsupported below. This is a big scary blob to have at the top of this file.

Reply:
Yeah I felt the same. Just promify where it is required directly.